### DISCUSSION TITLE
Kein Zugriff über internet

### DISCUSSION THREAD

**Discussion by tomgme** [brutella/hkknx-public#168] [General]
Bitte um Hilfe, und zwar. HomeKit und HKKNX funktionieren i WLAN Umgebung traumhaft ohne Macken,
Status:
KNX Gateway - Connected
HomeKit Bridge - Available
WPNS Server - Reachable

kann ich dafür seit zwei Wochen nicht über internet zugreifen... woran kann es liegen? gar nichts im Settings geändert... habe glaub ich mein server mal Updatet, debian server... 

Ideen?

**Comment by brutella** [comment]
> kann ich dafür seit zwei Wochen nicht über internet zugreifen... woran kann es liegen?

Der Zugriff vom Internet aus funktioniert über deine Steuerzentrale. Es gibt dazu eine Erklärung im Online Handbuch → https://hochgatterer.me/de/hkknx/docs/homekit/#fernzugriff-aktivieren

**Reply by tomgme** [reply]
Der HomePod mini war schuld. Vor etwa zwei Woche habe WLAN Einstellungen geändert und musste fast alle geräte nur mit WLAN verbinden.... Über HomePOD habe Vergessen. Hat zwar nicht so wie immer funktioniert aber... kein zeit gehabt. Jetzt geht alles wie gewöhnt, danke vielmals für die rasche Hilfe und support. Grüsse aus Poysbrunn
